README.adoc

```diff
@@ -145,306 +134,190 @@ The [hotspot=from file=0]`FROM` instruction initializes a new build stage and in
 image is built. If you don't need a parent image, then use `FROM scratch`, which makes your image a
 base image. 
 
-In this case, you're using the `openliberty/open-liberty:kernel-java8-openj9-ubi` image as your parent image, which comes with the latest Open Liberty runtime.
+In this case, you’re using the `openliberty/open-liberty:kernel-java8-openj9-ubi` image as your parent image, 
+which comes with the latest Open Liberty runtime.
 
-The [hotspot=link file=0]`RUN` instruction executes various shell commands in a new layer on top of the current image.
-In this case, you create a symlink between the `/opt/ol/wlp/usr/servers` directory and the `/servers`
-directory. This way, you can mount your servers more easily because you don't need to use long path names.
-Note that since the Open Liberty Docker image runs by default with user `1001` (which is a non-root user),
-you must temporarily switch to the `root` user to create the symlink. This is done by using 
-the [hotspot=user-root hotspot=user file=0]`USER` instruction. 
+The [hotspot=copy file=0]`COPY` instructions are structured as `COPY` 
+`[--chown=<user>:<group>]` `<source>` `<destination>`. 
+They copy local files into the specified destination within your Docker image.
+In this case, the server configuration file that is located at `src/main/liberty/config/server.xml` 
+is copied to the `/config/` destination directory.
 
-The [hotspot=entrypoint file=0]`ENTRYPOINT` and [hotspot=default-start]`CMD` instructions define a default command that executes when the image runs as
-a container. These two instructions function the same way, except that the [hotspot=default-start file=0]`CMD` instruction is overridden
-with any arguments that are passed at the end of the `docker run` command. In contrast, the
-[hotspot=entrypoint file=0]`ENTRYPOINT` instruction requires the `--entrypoint` flag to be overridden. In this case, you use
-the [hotspot=entrypoint file=0]`ENTRYPOINT` instruction to start an Open Liberty server and the [hotspot=default-start file=0]`CMD` instruction to indicate
-which server to start. Because the [hotspot=default-start file=0]`CMD` instruction is easily overridden, starting any server is convenient.
+=== Writing a .dockerignore file
 
-For a complete list of available instructions, see the https://docs.docker.com/engine/reference/builder[Docker documentation^].
+When Docker runs a build, it sends all of the files and directories that are
+located in the same directory as the Dockerfile to its build context, making
+them available for use in instructions like `ADD` and `COPY`. If there are files
+or directories you wish to exlcude one from the build context, you can add them
```

Comment:
Change "If there are files or directories you wish to exlcude one from the build context" to "If there are files
or directories you wish to exclude from the build context" to fix the typo errors.

Reply:
Fixed. Sorry about that.